### ISSUE TITLE
Double movement keys with VoiceOver?

### ISSUE THREAD

**Issue by jhersh** [splinesoft/MUDRammer#184]
> Mit Voiceover funktionieren die Bluetoothtasten fuer die Richtungen nicht richtig.. Die Nord-Sued-Tasten gehen, bei Westen wird immer zweimal ausgeführt. Wäre es möglich, die tasten für se, e und ne umzustellen, so dass man wahlweise o, so und no hätte?

> VoiceOver Bluetooth keyboard for the directions do not work properly .. The North-South keys go in the West is always executed twice. Would it be possible for the keys se, e and ne switch, so you could either o, and so no?

**Comment by jhersh** [comment]
> I have a iPHone 5 with IOS 8.2 and a Createc Bluetooth keyboard. 
I use voiceover, so I cannot use the movement control. 
I tried to use the ctrl-cursor keys for directions, 
but East and West commands always are executed two times, 
north and south work correctly, the rest doesn’t work at all. 
I cannot turn off voiceover, there is no speak any more if I try that. 
The second problem ist a differing language. 
instead of ne, e and se I need io, no and so. 
Best regards:

**Comment by jhersh** [comment]
> CTRL+Option is the main combination for nearly all VoiceOver commands. 
That means, That means every VoiceOver user will soon encounter this problem. 
It would be more than helpful to change to, let’s say
